docs/sources/flow/reference/components/mimir.rules.kubernetes.md

```diff
@@ -128,9 +128,12 @@ Name       | Type           | Description
 
 The `operator` argument should be one of the following strings:
 
-* `"in"`
-* `"notin"`
-* `"exists"`
+* `"In"`
+* `"NotIn"`
+* `"Exists"`
+* `"DoesNotExist"`
+
+When using the `operator` `"Exists"` or `"DoesNotExist"`, omit `values`.
```

Comment:
The sentence here reads a little weirdly to me personally, can we change it to something like:

```
The `values` argument must not be provided when `operator` is set to `"Exists"` or `"DoesNotExist"`.
```

cc @clayton-cornell for thoughts

Reply:
Yes, that's much better. I didn't want to tweak the text before because I wasn't sure if it could be (didn't want to change any technical meaning/intent)